requirements.txt

```diff
@@ -1,6 +1,6 @@
 bottleneck
 cftime
-dask
+dask==2021.02.0
```

Comment:
Don’t understand what’s the problem with the March version https://docs.dask.org/en/latest/changelog.html